src/lib/stores.ts

```diff
@@ -0,0 +1,4 @@
+import { writable } from 'svelte/store';
+import type { GridParams } from './types';
+
+export const gridSettings = writable<GridParams | undefined>(undefined);
```

Comment:
There is no need for store at the package level. Grid component is uses a buildin svelte context manager, which creates a separated store for each `<Grid />` instance.

Reply:
It doesnt need to accses store, just its value, which can be passed through constructor. And can be updated as simple as `$: controller.gridParams = $gridParams`

Reply:
got it! sent a new commit